coralproject/talk
# Fix duplicate reply notification emails

## What does this PR do?

Fixes receiving duplicate notifications of comment replies when a moderator approves a comment in post-moderation mode. See #3718.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

In an instance configured with post moderation, create a comment, and enable onReply notifications for that account. From a different (non-staff) account, reply to that comment. Observe that a notification is generated (either via having an SMTP test server configured, or by enabling digests for the replied-to user and checking the user's digests field in the database). In the moderation panel, accept the reply. Observe that an additional notification is _not_ generated.
 
## How do we deploy this PR?

No special actions required.
